core/templates/domain/question/question-backend-api.service.ts

```diff
@@ -61,6 +61,24 @@ export class QuestionBackendApiService {
     });
   }
 
+  private _fetchTotalQuestionCountForSkillIds(skillIds: Array<string>,
+      successCallback: (value?: Object | PromiseLike<Object>) => void,
+      errorCallback: (reason?: any) => void): void {
```

Comment:
why `any`?

/cc @nishantwrp

Reply:
I believe this is not any because we don't know the type, this is a valid ```any``` type case. The reason can be a string or an object etc.

Reply:
@Showtim3 you can write the type of argument, you are passing here. I think it should be `string` here?(because of errorResponse.error)

Reply:
Ok Noted. Will add this in a follow up PR.